en/messages.json

```diff
@@ -56,16 +56,15 @@
         "message": "Continue"
     },
     "popupRemainingAliases": {
```

Comment:
The ID needs to change when you change the string.

Short explanation: look at [German](https://pontoon.mozilla.org/de/firefox-relay-add-on/all-resources/?search=popupRemainingAliases&string=231315), nobody will know that the text changed, and the add-on will show a `$MAX`.

Long explanation: https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html

Reply:
Even better (fresh document which covers both issues in this PR, plurals and string change): https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#choose-self-explanatory-string-ids